Doc/howto/regex.rst

```diff
@@ -327,6 +329,12 @@ backslashes are not handled in any special way in a string literal prefixed with
 while ``"\n"`` is a one-character string containing a newline. Regular
 expressions will often be written in Python code using this raw string notation.
 
+In addition, special escape sequences that are valid in ``re``, but not valid
```

Comment:
Use `RE` or `regular expressions` instead of ` ``re`` `.

Reply:
Done